Packs/UrlScan/Integrations/UrlScan/Urlscan_test.py

```diff
@@ -29,3 +31,13 @@ def test_continue_on_blacklisted_error_arg(mocker, requests_mock, continue_on_bl
         assert return_error_mock.call_count == 0
     else:
         assert return_error_mock.call_count == 1
+
+
+def test_endless_loop_on_failed_response(requests_mock, mocker):
+    from UrlScan import format_results
+    mocker.patch(RETURN_ERROR_TARGET)
+
+    with open('./test_data/capitalne.json', 'r') as f:
+        response_data = json.loads(f.read())
+    requests_mock.get(RESULT_URL + 'uuid', status_code=200, json=response_data)
+    format_results('uuid')
```

Comment:
assert results

Reply:
All this method does is demisto_results, it does not return anything, not sure what to assert